content/en/docs/topics/advanced.md

```diff
@@ -183,3 +184,38 @@ access to Secret resources. For instance, the default
 configured for [encrypted storage](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).
 Please keep that in mind if you decide to switch to the ConfigMap backend, as it
 could expose your application's sensitive data.
+
+### SQL storage backend
+
+There is a ***beta*** SQL storage backend that stores release information in an SQL
+database.
+
+Using such a storage backend is particularly useful if your release information
+weighs more than 1MB (in which case, it can't be stored in ConfigMaps/Secrets
+because of internal limits in Kubernetes' underlying etcd key-value store).
+
+To enable the SQL backend, you'll need to deploy a SQL database and set the
+environmental variable `HELM_DRIVER` to `sql`. The DB details are set with the
+environmental variable `HELM_DRIVER_SQL_CONNECTION_STRING`.
+
+You can set it in a shell as follows:
+
+```shell
```

Comment:
We have an ongoing discussion around `console` vs `shell` in https://github.com/helm/helm-www/pull/744.

Reply:
Update once resolved, thanks @bridgetkromhout